### PR TITLE
#35: Make S3 region configurable (closes #35)

### LINKED ISSUE #35: Make S3 region configurable
## description
S3MailData is currently configured specifically for Frankfurt S3 region. This should be configurable.

## misc
https://github.com/buildo/mailo/blob/master/src/main/scala/mailo/data/S3MailData.scala#L41

### PR DESCRIPTION
**Pull request by calippo**
Closes #35

### tests performed
- tests should work properly

### CODE REVIEW

File: src/main/resources/application.conf.example [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
-  bucket = ""
-  partialsFolder = ""
-}
+  mailgun {
+    key = ""
+    uri = ""
+  }
 
+  cachingTTLSeconds = 30
+}
```
Comment (bytecodeguru) [inline src/main/resources/application.conf.example]: Missing EOL

**Review by bytecodeguru — CHANGES_REQUESTED** [review]
almost

**Comment by calippo** [comment]
@bytecodeguru fixed

**Review by bytecodeguru — COMMENTED** [review]
🍾 🍾 🍾

**Review by bytecodeguru — APPROVED** [review]

**Pull request merged**